dotnet-docs/token-authentication.md

```diff
@@ -0,0 +1,115 @@
+
+Each Token may be used to authenticate remotely against the Report Server for .NET with the Reading permissions of the corresponding User.
+The 'Report Access Tokens' view of the Report Server Manager shows the main properties of the Token:
+* The Token _Name_
+* When was the Token _Created_
+* When the Token _Expires_
+* When was the Token _Last Used_
+* Where was the _Last Connection From_ with the Token
+* The Token _Status_ You may control it through a switch in the view
+
+The view also lets you create a new Token through a button:
+
+![The view for creating a new personal access token for the logged-in user.](../images/rs-net-images/rs-net-token-view-create-access-token.png)
+
+The Token has an expiration time selected by the User while creating it:
+
+![Create a new personal access token for the logged-in user and select an expiration time from the available options (7-365 days).](../images/rs-net-images/rs-net-token-create-access-token.png)
+
+When the new Token is created, the UI lets you copy it from a dedicated form. You need to store it securely as it is not kept by the Report Server for .NET in raw format and cannot be restored later.
+
+![The form letting the user copy the newly created personal access token with the warning that the token should be stored securely as it will be shown only once.](../images/rs-net-images/rs-net-token-copy-access-token.png)
+
+Each `personalAccessToken` is passed from the Viewer to the Report Server for .NET only once per viewer's session, in the `PersonalToken` request. This ensures higher security in the communication between the viewer and the Report Server for .NET.
+
+The response from the server to the `PersonalToken` request contains two temporary tokens:
+* `accessToken` with a default lifespan of 30 minutes;
+* `refreshToken` with a default lifespan of 2 days.
+
+Both temporary tokens are kept in the browser cache for the viewer's session. This means that a refresh of the browser will reset the process, i.e., the `personalAccessToken` would be passed again to the server to receive a new temporary couple of tokens.
+
+![The response to the 'PersonalToken' request from the Report Server for .NET with a couple of temporary tokens.](../images/rs-net-images/rs-net-token-personaltoken-response.png)
+
+The `accessToken` is used by the viewer to authenticate the requests. When it expires, the viewer utilizes the `refreshToken` to receive another temporary couple of tokens, and so on:
+
+![The response to the 'refresh' request from the Report Server for .NET with the new couple of temporary tokens.](../images/rs-net-images/rs-net-token-refresh-response.png)
+
+The existing Tokens may be disabled and deleted by the user. In this case, all the access-refresh token couples created by this personal token become immediately invalid.
+
+If a User is deleted from the Report Server for .NET, all its Tokens become immediately invalid, along with the temporary token couples created with it.
```

Comment:
And when a user is disabled the tokens are not deleted but disabled. If a user is re-enabled, the tokens have to be manually enabled. (edit: I just saw that we have this mentioned for the guess user, so we just have to have it here too)

Reply:
The info is added